documentation/src/main/docs/faq.adoc

```diff
@@ -572,6 +572,19 @@ It gets the items from the lists and passes them downstream:
 include::plantuml/disjoint.puml[]
 ----
 
+`Multi.createBy().repeating()` lets you should the number of iteration using:
```

Comment:
Looks like this is missing something, "lets you should the number of iteration using" doesn't make sense

Reply:
should -> choose .... almost right :-)